### ISSUE TITLE
Standalone RTAB-Map source preference init failed

### ISSUE THREAD

**Issue by ziqB** [introlab/rtabmap#224]
Hi I'm currently trying to find out the frame rate of two cameras simultaneously. The model i'm using is the xtion pro live. I am only able to use one camera at a time if i open RTAB-map but that's not a problem i can always open another RTAB-map using a different user. What i've been trying to do is to connect a different camera on the each RTAB-map i have opened to check the frame rate. 

I went to Select sensor driver>More Options...
![more options](https://user-images.githubusercontent.com/29559599/28701073-42199928-7386-11e7-8f5d-b33ca3bdd876.png)
Here i found that you could put the device id. However I was not able to initialize it by putting the device id or its usb port location that i got from device manager not even for the camera that was already running. I have also noticed that most of the properties of the devices are the same. I'm not sure if it's because of the same properties both cameras have or that i did not put a valid id. Hope someone could enlighten me, I'm open to other methods to find out the frame rates of two cameras to play concurrently.

**Comment by matlabbe** [comment]
Yes the device ID would let you choose between the cameras (e.g., "#1", "#2" or "1", "2"). You can open Preferences->Logging and set logger level to Debug. On terminal when you start the camera, you will have more info about the camera initialization, if device is found or not. Camera initialization may also fail if the camera is already used by another application. Note also that each camera should be on its own USB hub, as starting two cameras on the same USB hub would fail.
